tests/src/Unit/AvPortalResourceTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\media_avportal\src\Unit;
+
+use Drupal\media_avportal\AvPortalResource;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Tests the AvPortalResource class.
+ */
+class AvPortalResourceTest extends UnitTestCase {
+
+  /**
+   * Tests the photo thumbnail preferred resolution.
```

Comment:
We actually test the whole `getPhotoThumbnailUrl()` method in here, so let's update the phpdoc.